src/ncp-spinel/SpinelNCPControlInterface.cpp

```diff
@@ -328,6 +329,39 @@ SpinelNCPControlInterface::joiner_add(
 	return;
 }
 
+void
+SpinelNCPControlInterface::joiner_attach(CallbackWithStatus cb)
+{
+	mNCPInstance->start_new_task(SpinelNCPTaskSendCommand::Factory(mNCPInstance)
+			.set_callback(cb)
+			.add_command(SpinelPackData(
+					SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_BOOL_S),
+					SPINEL_PROP_NET_STACK_UP,
+					true
+					))
+			.finish()
+			);
+}
+
+void
+SpinelNCPControlInterface::joiner_commission(
+		bool action,
+		const char *psk,
+		const char *provisioning_url,
+		CallbackWithStatus cb
+) {
+	syslog(LOG_NOTICE, "action %d psk %s provisioning_url %s", action, psk, provisioning_url);
```

Comment:
nit: adding "JoinerCommision: action %d..." in the log?

Reply:
how about removing this log, since we have similar one in `SpinelNCPTaskJoinerCommission`?

Reply:
forgot to remove the debugging information output. will remove it.